build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -0,0 +1,258 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Utility function to return the platform name in crosscompile.
+function(get_platform PLATFORM)
```

Comment:
might want to prefix this function name with `iree_`

Reply:
Done.